### PR TITLE
Add transactions support for removeByFind

### PR DESCRIPTION
**Pull request by rehmanvirani-10p**
Adds transaction support for removeByFind.

### CODE REVIEW

File: src/Model.js [hunk 1]
```diff
@@ -626,7 +626,9 @@ export class Model {
             throw new OneTableArgError('Remove cannot retry', {properties})
         }
         params.parse = true
-        let items = await this.find(properties, params)
+        let findParams = Object.assign({}, params);
+        delete findParams.transaction;
+        let items = await this.find(properties, findParams)
```
Comment (rehmanvirani-10p) [inline src/Model.js:631]: I was getting `Unknown transaction operation find` error because `removeByFind` is using the same params.

**Comment by mobsense** [comment]
Good catch. Thank you.

**Comment by rehmanvirani-10p** [comment]
@mobsense Thanks for accepting the PR. I see this change in the [v2.4.0](https://github.com/sensedeep/dynamodb-onetable/releases/tag/v2.4.0) tag and it's in pre-release status. Is there something blocking the official release?

**Comment by mobsense** [comment]
Our process is to accumulate changes under the next release tag and let folks test them out for a while before release.  There is nothing blocking release, but nothing forcing either. We try to balance between getting fixes out quickly and having too many releases.

Can you post here if the current repo 2.4 source is working well for you. That feedback helps accelerate the release.

**Pull request merged**